tests/hmm/hmumu_utils.py

```diff
@@ -92,7 +92,8 @@ def analyze_data(
     hrelresolution = None,
     zptreweighting = None,
     puidreweighting = None,
-    random_seed = 0,
+    btag_weights = None,
+    random_seed = 0 
```

Comment:
looks like you are missing a comma here, the automatic check that the code runs failed, see [here](https://travis-ci.com/jpata/hepaccelerate-cms/builds/137299416#L353). You can check this yourself if you click on the red X next to the pull request and go to "Build details".

Reply:
done